test/integration/upgrade-command/first-operator/templates/configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
I don't understand the changes here for `first-operator`

Reply:
this is because we can no longer test pod in integration test because that has health evaluated

Reply:
As @alenkacz said we abused the fact that our `health.go` would mark Pods healthy by default. This is not the case anymore.